tests/_test_msui/test_msui.py

```diff
@@ -328,3 +333,101 @@ def test_flight_track_io(self, mockload, mocksave, mockq, mocki, mockw):
         assert self.window.listFlightTracks.count() == 2
         assert os.path.exists(self.save_ftml)
         os.remove(self.save_ftml)
+
+
+class Test_MSUIMainWindow:
+    def test_storing(self, qtbot):
+        """Test the full scenario: create flight track, open TopView, modify settings,
+           save on close, and restore settings on reopen."""
+        window = msui_mw.MSUIMainWindow()
+        window.show()
+        qtbot.wait_exposed
```

Comment:
This is a method that you have to call with the widget to wait for as an argument.